# -*- coding: utf-8 -*-
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_NineStrip(object):
    def setupUi(self, NineStrip):
        NineStrip.setObjectName("NineStrip")
        NineStrip.setWindowFlags(QtCore.Qt.Window)
        NineStrip.setWindowModality(QtCore.Qt.NonModal)
        NineStrip.resize(618, 703)
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(NineStrip)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.TitleLabel = TitleLabel(NineStrip)
        self.TitleLabel.setMinimumSize(QtCore.QSize(0, 100))
        self.TitleLabel.setAlignment(QtCore.Qt.AlignCenter)
        self.TitleLabel.setProperty("pixelFontSize", 31)
        self.TitleLabel.setObjectName("TitleLabel")
        self.verticalLayout_2.addWidget(self.TitleLabel)
        self.horizontalLayout_4 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_4.setContentsMargins(-1, -1, 0, -1)
        self.horizontalLayout_4.setObjectName("horizontalLayout_4")
        self.reset_btn = PushButton(NineStrip)
        self.reset_btn.setMaximumSize(QtCore.QSize(100, 16777215))
        self.reset_btn.setObjectName("reset_btn")
        self.horizontalLayout_4.addWidget(self.reset_btn)
        self.verticalLayout_2.addLayout(self.horizontalLayout_4)
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setSpacing(0)
        self.verticalLayout.setObjectName("verticalLayout")
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setSpacing(0)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.btn11 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn11.sizePolicy().hasHeightForWidth())
        self.btn11.setSizePolicy(sizePolicy)
        self.btn11.setObjectName("btn11")
        self.horizontalLayout.addWidget(self.btn11)
        self.line_5 = QtWidgets.QFrame(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.line_5.sizePolicy().hasHeightForWidth())
        self.line_5.setSizePolicy(sizePolicy)
        self.line_5.setMaximumSize(QtCore.QSize(7, 16777215))
        self.line_5.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_5.setLineWidth(7)
        self.line_5.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_5.setObjectName("line_5")
        self.horizontalLayout.addWidget(self.line_5)
        self.btn12 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn12.sizePolicy().hasHeightForWidth())
        self.btn12.setSizePolicy(sizePolicy)
        self.btn12.setObjectName("btn12")
        self.horizontalLayout.addWidget(self.btn12)
        self.line_6 = QtWidgets.QFrame(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.line_6.sizePolicy().hasHeightForWidth())
        self.line_6.setSizePolicy(sizePolicy)
        self.line_6.setMaximumSize(QtCore.QSize(7, 16777215))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.line_6.setFont(font)
        self.line_6.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_6.setLineWidth(10)
        self.line_6.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_6.setObjectName("line_6")
        self.horizontalLayout.addWidget(self.line_6)
        self.btn13 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn13.sizePolicy().hasHeightForWidth())
        self.btn13.setSizePolicy(sizePolicy)
        self.btn13.setObjectName("btn13")
        self.horizontalLayout.addWidget(self.btn13)
        self.verticalLayout.addLayout(self.horizontalLayout)
        self.line = QtWidgets.QFrame(NineStrip)
        self.line.setMinimumSize(QtCore.QSize(0, 7))
        self.line.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line.setLineWidth(10)
        self.line.setFrameShape(QtWidgets.QFrame.HLine)
        self.line.setObjectName("line")
        self.verticalLayout.addWidget(self.line)
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_2.setSpacing(0)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.btn21 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn21.sizePolicy().hasHeightForWidth())
        self.btn21.setSizePolicy(sizePolicy)
        self.btn21.setObjectName("btn21")
        self.horizontalLayout_2.addWidget(self.btn21)
        self.line_7 = QtWidgets.QFrame(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.line_7.sizePolicy().hasHeightForWidth())
        self.line_7.setSizePolicy(sizePolicy)
        self.line_7.setMaximumSize(QtCore.QSize(7, 16777215))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.line_7.setFont(font)
        self.line_7.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_7.setLineWidth(10)
        self.line_7.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_7.setObjectName("line_7")
        self.horizontalLayout_2.addWidget(self.line_7)
        self.btn22 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn22.sizePolicy().hasHeightForWidth())
        self.btn22.setSizePolicy(sizePolicy)
        self.btn22.setObjectName("PushButton_22")
        self.horizontalLayout_2.addWidget(self.btn22)
        self.line_8 = QtWidgets.QFrame(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.line_8.sizePolicy().hasHeightForWidth())
        self.line_8.setSizePolicy(sizePolicy)
        self.line_8.setMaximumSize(QtCore.QSize(7, 16777215))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.line_8.setFont(font)
        self.line_8.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_8.setLineWidth(10)
        self.line_8.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_8.setObjectName("line_8")
        self.horizontalLayout_2.addWidget(self.line_8)
        self.btn23 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn23.sizePolicy().hasHeightForWidth())
        self.btn23.setSizePolicy(sizePolicy)
        self.btn23.setObjectName("PushButton_23")
        self.horizontalLayout_2.addWidget(self.btn23)
        self.verticalLayout.addLayout(self.horizontalLayout_2)
        self.line_4 = QtWidgets.QFrame(NineStrip)
        self.line_4.setMinimumSize(QtCore.QSize(0, 7))
        self.line_4.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_4.setLineWidth(10)
        self.line_4.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_4.setObjectName("line_4")
        self.verticalLayout.addWidget(self.line_4)
        self.horizontalLayout_3 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_3.setSpacing(0)
        self.horizontalLayout_3.setObjectName("horizontalLayout_3")
        self.btn31 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn31.sizePolicy().hasHeightForWidth())
        self.btn31.setSizePolicy(sizePolicy)
        self.btn31.setObjectName("btn31")
        self.horizontalLayout_3.addWidget(self.btn31)
        self.line_9 = QtWidgets.QFrame(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.line_9.sizePolicy().hasHeightForWidth())
        self.line_9.setSizePolicy(sizePolicy)
        self.line_9.setMaximumSize(QtCore.QSize(7, 16777215))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.line_9.setFont(font)
        self.line_9.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_9.setLineWidth(10)
        self.line_9.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_9.setObjectName("line_9")
        self.horizontalLayout_3.addWidget(self.line_9)
        self.btn32 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn32.sizePolicy().hasHeightForWidth())
        self.btn32.setSizePolicy(sizePolicy)
        self.btn32.setObjectName("btn32")
        self.horizontalLayout_3.addWidget(self.btn32)
        self.line_10 = QtWidgets.QFrame(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.line_10.sizePolicy().hasHeightForWidth())
        self.line_10.setSizePolicy(sizePolicy)
        self.line_10.setMaximumSize(QtCore.QSize(7, 16777215))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.line_10.setFont(font)
        self.line_10.setFrameShadow(QtWidgets.QFrame.Plain)
        self.line_10.setLineWidth(10)
        self.line_10.setFrameShape(QtWidgets.QFrame.VLine)
        self.line_10.setObjectName("line_10")
        self.horizontalLayout_3.addWidget(self.line_10)
        self.btn33 = PushButton(NineStrip)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.btn33.sizePolicy().hasHeightForWidth())
        self.btn33.setSizePolicy(sizePolicy)
        self.btn33.setObjectName("btn33")
        self.horizontalLayout_3.addWidget(self.btn33)
        self.verticalLayout.addLayout(self.horizontalLayout_3)
        self.verticalLayout_2.addLayout(self.verticalLayout)

        self.retranslateUi(NineStrip)
        QtCore.QMetaObject.connectSlotsByName(NineStrip)

    def retranslateUi(self, NineStrip):
        _translate = QtCore.QCoreApplication.translate
        NineStrip.setWindowTitle(_translate("NineStrip", "Form"))
        self.TitleLabel.setText(_translate("NineStrip", "红方Vs蓝方"))
        self.reset_btn.setText(_translate("NineStrip", "重开"))
        self.btn11.setText(_translate("NineStrip", ""))
        self.btn12.setText(_translate("NineStrip", ""))
        self.btn13.setText(_translate("NineStrip", ""))
        self.btn21.setText(_translate("NineStrip", ""))
        self.btn22.setText(_translate("NineStrip", ""))
        self.btn23.setText(_translate("NineStrip", ""))
        self.btn31.setText(_translate("NineStrip", ""))
        self.btn32.setText(_translate("NineStrip", ""))
        self.btn33.setText(_translate("NineStrip", ""))
from qfluentwidgets import PushButton, TitleLabel, Dialog
